doc/locale/ja/LC_MESSAGES/news.po

```diff
@@ -362,12 +566,12 @@ msgid ""
 "for synonyms are many, we are troublesome to define synonyms. Because we "
 "need to define many similar definitions."
 msgstr ""
-"多くのケースでは、 ``mroonga`` を ``mroonga OR tritonn OR \"groonga mysql"
-"\"`` と展開した場合、 ``tritonn`` や ``\"groonga mysql\"`` も ``mroonga OR "
-"tritonn OR \"groonga mysql\"`` と展開して欲しくなりますが、今までは、そのよう"
-"なケースでは定義を追加する必要がありました。したがって、対象のキーワードに同"
-"義語がたくさんあると、同じような定義をたくさん定義する必要があるため、同義語"
-"の定義が煩雑でした。"
+"多くのケースでは、 ``mroonga`` を ``mroonga OR tritonn OR \"groonga "
+"mysql\"`` と展開した場合、 ``tritonn`` や ``\"groonga mysql\"`` も ``mroonga "
+"OR tritonn OR \"groonga mysql\"`` と展開して欲しくなりますが、今までは、その"
+"ようなケースでは定義を追加する必要がありました。したがって、対象のキーワード"
+"に同義語がたくさんあると、同じような定義をたくさん定義する必要があるため、同"
+"義語の定義が煩雑でした。"
```

Comment:
This modification does not related release note for 13.0.1.

Reply:
I'm thinking about diffs that automatically created by `make`...
I will revert these unrelated changes, but... should they updated?
Could we update these in another pull request?

Reply:
Revert changes.